themes/digital.gov/src/scss/new/_article.scss

```diff
@@ -2,120 +2,116 @@
 
 // Article
 
-main{
-	@include u-padding-y(3);
-	@include at-media('tablet') {
-		@include u-padding-y(4);
-	}
+main {
+  // Base Article Header Styles
+  header.page-header,
+  > header,
+  > article > header {
+    @include u-padding-bottom(2);
+    @include u-margin-bottom(3);
 
-	// Base Article Header Styles
-	header.page-header,
-	> header,
- 	> article > header{
-		@include u-padding-bottom(2);
-		@include u-margin-bottom(3);
+    // Breadcrumbs
+    .breadcrumb {
+      @include u-margin-top(4);
+      @include u-margin-bottom(2);
+      @include u-text("bold");
+      @include u-font("sans", "md");
+	  @include at-media("tablet") {
+		@include u-margin-top(6);
+	  }
+      a {
+        span {
+          @include u-padding-bottom("2px");
+        }
+        @include u-text("no-underline");
+        &:hover span {
+          border-bottom: 3px #656bd7 solid; // $indigo-40
+        }
+      }
+    }
```

Comment:
How does this differ from `themes/digital.gov/src/scss/new/_breadcrumbs.scss`?

In general overrides should be minimal. We can refactor this later, just bringing it up for awareness.